aces/Loris
# [Candidate Parameters] Updated help text

Updated the help text for the Candidate Parameters page

#### Testing instructions

1. Edit copy if necessary
2. Check screenshot for rendering

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576#issuecomment-556905146

![Screenshot 2019-11-24 18 27 54](https://user-images.githubusercontent.com/37813606/69503481-2984ec00-0ee8-11ea-84bf-3e8e63e6018a.png)
